docs/reference/release-notes/7.12.asciidoc

```diff
@@ -1,3 +1,109 @@
+* Lazily load soft-deletes for searchable snapshot shards {es-pull}69203[#69203]
+
+
+
+[[bug-7.12.1]]
+[float]
+=== Bug fixes
+
+Aggregations::
+* Fix percentiles agg in slow log after transport {es-pull}70318[#70318]
+
+Analysis::
+* Fix position increment gap on phrase/prefix analyzers {es-pull}70096[#70096] (issue: {es-issue}70049[#70049])
+
+CRUD::
+* Fork listener#onFailure in `PrimaryReplicaSyncer` {es-pull}70506[#70506] (issues: {es-issue}69949[#69949], {es-issue}70407[#70407])
+
+Cluster Coordination::
+* Fully initialize cluster state on ephemeral nodes {es-pull}71466[#71466]
+
+Features/Features::
+* Map data tiers roles onto DATA legacy role for <7.3 {es-pull}71628[#71628] (issue: {es-issue}71464[#71464])
+
+Features/ILM+SLM::
+* Delete data stream in ILM delete action if last index in data stream {es-pull}69637[#69637]
+
+Features/Ingest::
+* Fix handling of non-integer port values in `community_id` processor {es-pull}70148[#70148] (issue: {es-issue}70131[#70131])
+* Fix typo in validation for destination port of community ID processor {es-pull}70883[#70883]
+* Templates match indices with date math expressions {es-pull}71433[#71433] (issue: {es-issue}69727[#69727])
+* URI parts processor handles URLs containing spaces {es-pull}71559[#71559] (issue: {es-issue}70947[#70947])
+
+Features/Stats::
+* Avoid return negative value in `CounterMetric` {es-pull}71446[#71446] (issues: {es-issue}52411[#52411], {es-issue}70968[#70968])
+
+Features/Watcher::
+* Enable Setting Master Node Timeout in Watcher Start/Stop Requests {es-pull}70425[#70425]
+
+Geo::
+* Do not over-allocate when resizing in `GeoGridTiler` {es-pull}70159[#70159]
+* Fix `geo_line` agg behavior with missing values {es-pull}69395[#69395] (issue: {es-issue}69346[#69346])
+* Fix infinite loop when polygonizing a circle with centre on the pole {es-pull}70875[#70875]
+* Fix overflow bug in `SortingNumericDocValues` {es-pull}70154[#70154]
+
+Infra/Core::
+* Always wrap date parsing exception into `IllegalArgumentException` {es-pull}71038[#71038]
+
+Infra/Scripting::
+* Remove loop counter for foreach loops {es-pull}71602[#71602] (issue: {es-issue}71584[#71584])
+
+Machine Learning::
+* Make ML memory tracker more robust to flipping on/off master nodes {es-pull}71067[#71067] (issue: {es-issue}68685[#68685])
+
+Mapping::
+* Legacy geo-shape mapper not detecting [points_only] parameter {es-pull}70765[#70765] (issue: {es-issue}70751[#70751])
+
+SQL::
+* Enforce and document dedicated client version compatibility {es-pull}70451[#70451] (issue: {es-issue}70400[#70400])
+* Fix manifest version tag in Tableau connector {es-pull}71524[#71524]
+* Resolve attributes recursively for improved subquery support {es-pull}69765[#69765] (issues: {es-issue}2[#2], {es-issue}4[#4], {es-issue}6[#6], {es-issue}7[#7], {es-issue}8[#8], {es-issue}22[#22], {es-issue}67237[#67237])
```

Comment:
I think the script was erroneously pulling these extra issues from the code example:
```suggestion
* Resolve attributes recursively for improved subquery support {es-pull}69765[#69765] (issue:  {es-issue}67237[#67237])
```